### PR TITLE
Wait for server start before running tests

### PR DESCRIPTION
**Pull request by andrew-nowak**
Resolves #NUMBER

**Overall change:** Attempts to resolve the intermittent errors when running `bbc-a11y` tests.

**Code changes:**

- ~~The version of `bbc-a11y` to be used is specified in *two* places in `package.json` -- in the devDependencies and in `test:prod:ci`. This is to avoid having to install the entirety of the devDependencies in a 'production tests' run. However, these two values can very easily fall out of sync (and already have!). Instead, *always request the latest*.~~
- The intermittent bbc-a11y failures seem to be due to a race condition between the server start and the first tests starting; use the `wait-on` script to try and avoid that race condition.

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

**Review by DemonusPC — DISMISSED** [review]
Dank

**Review by chris-hinds — COMMENTED** [review]
Nice work around. 👍

**Comment by staylos92** [comment]
No formal testing required for this.

File: package.json [hunk 1]
```diff
@@ -141,7 +141,7 @@
     "@storybook/react": "^5.0.11",
     "amphtml-validator": "^1.0.23",
     "babel-eslint": "^10.0.0",
-    "bbc-a11y": "^2.3.0",
+    "bbc-a11y": "latest",
```
Comment (jamesbhobbs) [inline package.json]: I don't think we should take beyond this major version, have discussed with @hindsc52

Comment (chris-hinds) [reply package.json]: Discussing this with @benjaminhobbs just now, he is right. We shouldn't be blindly installing the latest version of this package as we could very easily be introducing braking changes.

Comment (andrew-nowak) [reply package.json]: fair enough :+1:

**Review by rossgaskell — APPROVED** [review]
👍 LGTM

**Review by j-pendlebury — APPROVED** [review]
LGTM 👍

**Pull request merged**